dali/c_api/c_api.cc

```diff
@@ -212,6 +208,16 @@ void daliCreatePipeline(daliPipelineHandle *pipe_handle, const char *serialized_
                       cpu_prefetch_queue_depth, gpu_prefetch_queue_depth, enable_memory_stats);
 }
 
+inline std::unique_ptr<DALIPipeline> WrapPipeline(std::unique_ptr<dali::Pipeline> pipeline) {
+  auto pipe_wrap = std::make_unique<DALIPipeline>();
```

Comment:
How about making this `static`? Or a part of anonymous namespace above?

Reply:
Isn't it all equivalent?

Reply:
Moved.